### ISSUE TITLE
Think about decisions based on our Bayesian estimates

### ISSUE THREAD

**Issue by fbartolic** [fbartolic/hierarchical_single_lens_events#3]
The whole point of this thing is to use Bayesian predictions in order to make decisions which maximize planet yield (or something else?). We need to think about which expectation values do we want to compute, peak magnitude, magnification, or something else.

**Comment by astrophpeter** [comment]
I think maximizing planet yield is a good motivation but no the only one. As @nrgolovich mentioned follow up of potential black hole candidates (long events that show annual parallax and possible astrometric signal) is also very interesting. We want to follow up events that are going to show an anomaly that allows us to study them.

I think we should be computing expectations of magnification and timescale because these are good indicators of whether or not we are going to see a degeneracy-breaking anomaly in the future?

**Comment by nrgolovich** [comment]
For black hole early warning, the thought process goes like this:
In order to be confident you have a black hole you need astrometric microlensing in order to break the mass/distance degeneracy. That signal is noisy and hard to measure, so you have to get it at peak, which occurs a little after the photometric peak. The purest photometric cut you can make is a cut on high tE and low piE (paper proving this out is forthcoming soon). Unfortunately in order to make that selection you have to wait until after the photometric signal has fallen (at which time it's too late to measure the largest astrometric signal). So, how well can we estimate a small piE/large tE signal with just the rise of the light curve? That'll be necessary to get the most complete black hole candidate list for (expensive) astrometric follow up.

**Comment by fbartolic** [comment]
That is probably a better use case for the predictive model than planet hunting these days. @nrgolovich  do you know which datasets would be most useful for training the model in that case?

**Comment by nrgolovich** [comment]
@fbartolic 

OGLE has published light curves (beyond those in the EWS) for both OGLE 3 and OGLE 4. I've been using those for parallax fits. One thing about this task though is that stellar variability kills us. We'll need to fold in GP correlated noise modeling if we hope to do it well enough. If I remember right from your talk in NY, you've achieve this at least somewhat, right?

Data: 

Ogle 3: http://www.astrouw.edu.pl/ogle/ogle3/blg_tau/ 
Ogle 4: http://www.astrouw.edu.pl/ogle/ogle4/microlensing_maps/ 

Both are public. The drawback is OGLE is notoriously closed about their analysis choices, and their papers are not entirely sufficient to understand their cuts as far as I've been able to tell.

**Comment by fbartolic** [comment]
>One thing about this task though is that stellar variability kills us. We'll need to fold in GP correlated noise modeling if we hope to do it well enough. If I remember right from your talk in NY, you've achieve this at least somewhat, right?

Yes, `caustic` supports  GPs via celerite. Check out the tutorial [here](https://caustic.readthedocs.io/en/latest/examples/gaussian_processes.html). You can use GPs with parallax included, the only issue is [multimodality](https://caustic.readthedocs.io/en/latest/examples/annual_parallax.html) but Nested Sampling works alright. 

For prediction within a survey we don't need to understand their cuts I think.

**Comment by nrgolovich** [comment]
Yeah nice notebooks and tutorials. You’ve done a lot of work! I have proved out your results in my own experience with exoplanet/pymc3 parallax fits. I haven’t folded in GPs though, and they are definitely necessary.
